src/commands/api/request/rest.ts

```diff
@@ -36,18 +36,24 @@ export class Rest extends SfCommand<void> {
       char: 'H',
       multiple: true,
     }),
+    file: Flags.file({
+      summary: messages.getMessage('flags.file.summary'),
+      helpValue: 'file',
+      char: 'f',
```

Comment:
you can pass `exists` to have oclif validate the file:
https://github.com/oclif/core/blob/f38f8270bafa04cdcb8e6d193b808e099e205864/src/flags.ts#L121

also, we haven't been consistent with this but since this command already has `--stream-to-file`, what do you think about using `--request-file` instead?

https://github.com/search?q=org%3Asalesforcecli+%2FFlags%5C.file%5C%28%2F&type=code
https://github.com/salesforcecli/cli/wiki/Design-Guidelines-Flags